.changeset/curvy-toys-tap.md

```diff
@@ -0,0 +1,9 @@
+---
+"@hopper-ui/styled-system": patch
+"@hopper-ui/tokens": patch
+---
+
+Tokens: 
```

Comment:
you can remove the Tokens section, as this applies to both tokens and the styled-system